package.json

```diff
@@ -14,13 +14,14 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "build": "npm-run-all install:playground build:sass build:website build:playground build:minify-images",
+    "build": "npm-run-all install:playground build:sass build:website build:playground build:minify-images build:sponsors",
```

Comment:
This isn't actually part of the site, so I don't think it should be included in this command.

Reply:
sure